Fix/validations store defaults

jcampbell opened the pull request.

jcampbell committed:
WIP Updates to SiteBuilder Stores

jcampbell committed:
Merge remote-tracking branch 'origin/v0.8.x' into fix/validations_store_defaults

jcampbell committed:
Updates to SiteBuilder Stores

pep8speaks commented:
Thank you for submitting to Great Expectations!

















* In the file [`great_expectations/render/renderer/site_builder.py`](https://github.com/great-expectations/great_expectations/blob/3a9b95226aa423cc28404d34a2ab4527368e84ec/great_expectations/render/renderer/site_builder.py):

> [Line 297:9](https://github.com/great-expectations/great_expectations/blob/3a9b95226aa423cc28404d34a2ab4527368e84ec/great_expectations/render/renderer/site_builder.py#L297): [E128](https://duckduckgo.com/?q=pep8%20E128) continuation line under-indented for visual indent

















This linter comment has been updated based on the most recent commit.

##### Comment last updated at 2019-10-07 17:08:05 UTC

jcampbell committed:
PEP8 Speaks

jcampbell committed:
Update configuration and tests
  - Tests updated to new SiteBuilder configuration
  - Refactor name to InMemoryEvaluationParameterStore

jcampbell committed:
Linting updates

Aylr commented on great_expectations/data_context/templates.py:
```diff
@@ -97,51 +97,18 @@
+    class_name: InMemoryEvaluationParameterStore
 
 data_docs_sites:
   # Data Docs make it simple to visualize data quality in your project. These
   # include Expectations, Validations & Profiles. The are built for all
   # Datasources from JSON artifacts in the local repo including validations &
   # profiles from the uncommitted directory. Read more at https://docs.greatexpectations.io/en/""" + rtd_url_ge_version + """/features/data_docs.html
-  local_site: # site name
-    datasource_whitelist: '*' # used to restrict the Datasources
-    module_name: great_expectations.render.renderer.site_builder
+  local_site:
     class_name: SiteBuilder
-    target_store_name: local_site_html_store   
-    site_index_builder:
-      class_name: DefaultSiteIndexBuilder
-    site_section_builders:
-      expectations:
-        class_name: DefaultSiteSectionBuilder
-        source_store_name: expectations_store
-        renderer:
-          module_name: great_expectations.render.renderer
-          class_name: ExpectationSuitePageRenderer
-
-      validations:
-        class_name: DefaultSiteSectionBuilder
-        source_store_name: validations_store
-        run_id_filter:
-          ne: profiling
-        renderer:
-          module_name: great_expectations.render.renderer
-          class_name: ValidationResultsPageRenderer
-
-      profiling:
-        class_name: DefaultSiteSectionBuilder
-        source_store_name: validations_store
-        run_id_filter:
-          eq: profiling
-        renderer:
-          module_name: great_expectations.render.renderer
-          class_name: ProfilingResultsPageRenderer
```
I ❤️ a good red party ❌. This is so so so so great.

Aylr commented on tests/data_context/fixtures/post_init_project_v0.8.0_A/great_expectations/great_expectations.yml:
```diff
@@ -68,13 +68,7 @@ stores:
 #      key_prefix:
 
   evaluation_parameter_store:
-    module_name: great_expectations.data_context.store
-    class_name: EvaluationParameterStore
-
-  local_site_html_store:
-    module_name: great_expectations.data_context.store
-    class_name: HtmlSiteStore
-    base_directory: uncommitted/data_docs/local_site/
```
Nitpick: There are a pile of other fixture templates that _might_ have keys that could be removed.

Aylr reviewed (approved):
lgtm
